static/updatePanel.sh

```diff
@@ -46,8 +46,8 @@ echo "Panel is now in maintenance mode."
 db_connection=$(grep "^DB_CONNECTION=" "$env_file" | cut -d '=' -f 2)
 
 if [ -z "$db_connection" ]; then
-  echo "DB_CONNECTION not found in $env_file."
-  exitInstall 0
+  db_connection='sqlite'
+  echo "DB_CONNECTION not found in $env_file using $db_connection as default."
 fi
 
 echo "DB_CONNECTION is set to: $db_connection"
```

Comment:
Just a small nitpick, but can you put the other echo inside an `else`? Otherwise it prints twice.